### PR TITLE
Updated Puppeteer readme

### PR DESCRIPTION
**Pull request by executeautomation**
## Description
Updated the readme with the usage of the puppeteer plugin

## Server Details
NA

## Motivation and Context
The detail was missing and needed for beginners.

## How Has This Been Tested?
Tested in my local machine and it works

## Breaking Changes
NA

## Types of changes
Just readme file change

## Checklist

- [ y] I have read the [MCP Protocol Documentation](https://modelcontextprotocol.io)
- [ ] My server follows MCP security best practices
- [ ] I have updated the server's README accordingly
- [ ] I have tested this with an LLM client
- [ ] My code follows the repository's style guidelines
- [ ] New and existing tests pass locally
- [ ] I have added appropriate error handling
- [ ] I have documented all environment variables and configuration options

### CODE REVIEW

File: src/puppeteer/README.md [hunk 1]
```diff
@@ -52,6 +52,19 @@ The server provides access to two types of resources:
...
 
+## Configuration to use Puppeteer Server
+Here's the Claude Desktop configuration to use the Puppeter server:
+
+```json
+{
+  "mcpServers": {
+    "puppeteer": {
+      "command": "npx",
+      "args": ["-y", "@modelcontextprotocol/server-puppeteer", "http://example.com"]
```
Comment (sean-lynch) [inline src/puppeteer/README.md:63]: I don't think puppeteer server takes a URL in the args. It's passed in by calling the exposed tool functions. 

Seems like you should drop the third `"http://example.com"` arg

**Review by jspahrsummers — CHANGES_REQUESTED** [review]
Thanks for the PR! The comment is exactly right—please remove the extra arg, and we can get this merged in.

File: src/puppeteer/README.md (hunk 1 above)
Comment (executeautomation) [reply src/puppeteer/README.md]: Yes updated the same, thanks for the comments @sean-lynch !

**Review by jspahrsummers — APPROVED** [review]
Great, thank you!

**Pull request merged**